mne/conftest.py

```diff
@@ -140,6 +140,10 @@ def pytest_configure(config):
     ignore:Implementing implicit namespace packages.*:DeprecationWarning
     ignore:Deprecated call to `pkg_resources.*:DeprecationWarning
     ignore:pkg_resources is deprecated as an API.*:DeprecationWarning
+    # TODO: This one we should fix with .item()s
+    ignore:Conversion of an array with ndim > 0 to a scalar.*:DeprecationWarning
```

Comment:
is this now unnecessary?  I see a lot of `.item()` additions in the diff.

Reply:
I got tired of trying to get rid of them all but I'm working at it again :)